speech/src/main/java/com/example/speech/TranscribeStreamingV2.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_to_text_transcribe_streaming_v2]
+
+import com.google.api.gax.rpc.BidiStream;
+import com.google.cloud.speech.v2.*;
+import com.google.protobuf.ByteString;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class TranscribeStreamingV2 {
+    private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+
+    //    Transcribes audio from an audio file stream using Google Cloud Speech-to-Text API.
+//    Args:
+//        streamFile (String): Path to the local audio file to be transcribed.
+//            Example: "resources/audio.wav"
+//    Returns:
+//        List<StreamingRecognizeResponse>: A list of objects.
+//            Each response includes the transcription results for the corresponding audio segment.
+//
+    public static List<StreamingRecognizeResponse> transcribeStreamingV2(String streamFile) throws IOException {
+        List<StreamingRecognizeResponse> responses = new ArrayList<>();
+//      Instantiates a client
+        try (SpeechClient client = SpeechClient.create()) {
+
+            Path path = Paths.get(streamFile);
+            byte[] audioContent = Files.readAllBytes(path);
+
+//          In practice, stream should be a generator yielding chunks of audio data
+            int chunkLength = audioContent.length / 5;
+            List<byte[]> stream = new ArrayList<>();
+            for (int i = 0; i < audioContent.length; i += chunkLength) {
+                int end = Math.min(i + chunkLength, audioContent.length);
+                byte[] chunk = new byte[end - i];
+                System.arraycopy(audioContent, i, chunk, 0, end - i);
+                stream.add(chunk);
+            }
+
+
+            List<StreamingRecognizeRequest> audioRequests = new ArrayList<>();
+            for (byte[] audio : stream) {
+                audioRequests.add(StreamingRecognizeRequest.newBuilder().setAudio(ByteString.copyFrom(audio)).build());
```

Comment:
issue: break the builder call out onto its own line. Pass the build object into the call to `audioRequests.add()`.